### PR TITLE
Let's make a push to master-stable with this new format

### PR DESCRIPTION
**Pull request by ryelo**
Patch notes since last master-stable push:

* updating the welcome/quick start stuff with Yash findings

* chrome bullet points

* Add a bunch of UX stories

### CODE REVIEW

File: src/docs/welcome/quickStart/DOC.md [hunk 1]
```diff
@@ -89,31 +94,31 @@ Bootstrapped application with essential tools for rapid development
 1. Run the container with default options
 
     ```sh
-    bash scripts/run.sh
+    SPANDX_CONFIG=path/to/insights-frontend-starter-app/config/spandx.config.js sh path/to/insights-proxy/scripts/run.sh
```
Comment (iphands) [inline src/docs/welcome/quickStart/DOC.md:97]: Hey this is not a valid path anymore right?
landing-page-frontend/proxy-config/local-frontend.js

Is the new path it seems.

File: src/docs/welcome/quickStart/DOC.md [hunk 2]
```diff
@@ -89,31 +94,31 @@ Bootstrapped application with essential tools for rapid development
...
     ```
 
-#### More options for running
+##### More options for running
 
 * Run with the local Legacy Insighs API
```
Comment (iphands) [inline src/docs/welcome/quickStart/DOC.md]: "local Legacy Insighs API" we dont want to document this anymore I think right?
Also Insights is misspelled

File: src/docs/welcome/quickStart/DOC.md [hunk 3]
```diff
@@ -89,31 +94,31 @@ Bootstrapped application with essential tools for rapid development
...
 
     ```sh
-    LOCAL_API=true bash scripts/run.sh
+    SPANDX_CONFIG=path/to/insights-frontend-starter-app/config/spandx.config.js sh path/to/insights-proxy/scripts/run.sh LOCAL_API=true
```
Comment (iphands) [inline src/docs/welcome/quickStart/DOC.md]: Running with chrome is buried in the docs right?
One thing I see tons of teams doing is running chrome locally when they dont need to.
The Chrome docs are really for us, perhaps we should just put them in a separate area/file.

**Review by iphands — DISMISSED** [review]

File: src/docs/welcome/quickStart/DOC.md (hunk 1 above)
Comment (ryelo) [reply src/docs/welcome/quickStart/DOC.md:97]: I'm not sure why the landing page uses that path, but in the starter app it's:
`config/spandx.config.js`

https://github.com/RedHatInsights/insights-frontend-starter-app/blob/master/config/spandx.config.js

File: src/docs/welcome/quickStart/DOC.md (hunk 2 above)
Comment (ryelo) [reply src/docs/welcome/quickStart/DOC.md]: Removing

File: src/docs/welcome/quickStart/DOC.md (hunk 3 above)
Comment (ryelo) [reply src/docs/welcome/quickStart/DOC.md]: I'll just remove them for now.

**Review by karelhala — DISMISSED** [review]

**Review by kruai — DISMISSED** [review]

**Review by karelhala — DISMISSED** [review]

**Comment by ryelo** [comment]
@karelhala @kruai @iphands can we re-review?

**Review by karelhala — DISMISSED** [review]

**Review by kruai — DISMISSED** [review]

**Review by karelhala — DISMISSED** [review]

File: src/docs/uxd/actionsOnSystems/DOC.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+![Inventory view with inline global actions](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/inventoryInlineGlobal.png?raw=true)
+*Inventory view of a system, showing inline global actions*
+
+![Inventory view with service-level action](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/inventoryServiceLevel.png?raw=true)
+*Inventory view of a system, including service-level action (Remediate with Ansible)*
+
+## Considerations
+
+- Actions button would be there if ANY of the services have actions.
+- Entitlements & the actions button - what if they don't have access to a service that has actions (Solution: hid action button unless there are valid actions).
```
Comment (kruai) [inline src/docs/uxd/actionsOnSystems/DOC.md]: Suggestion: `Entitlements & the actions button - what if they don't have access to a service that has actions*?* (Solution: *hide* action button unless there are valid actions.)`

File: src/docs/uxd/actionsOnSystems/DOC.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+- Need a reliable way to get from a system to the inventory view of a system (to manage the global actions). (Link?)
```
Comment (kruai) [inline src/docs/uxd/actionsOnSystems/DOC.md]: Is `(Link?)` implying a TODO or is that intentional?

File: src/docs/uxd/actionsOnSystems/DOC.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+  - Tabs would force the global actions to be in the current "Actions" dropdown and app specific actions would live inside their respective tab area
```
Comment (kruai) [inline src/docs/uxd/actionsOnSystems/DOC.md]: Should be `app-specific` actions

File: src/docs/uxd/alertsNotifications/DOC.md [hunk 7]
```diff
@@ -1,5 +1,87 @@
...
+## Modal dialog processes
+
+After completing an action or process in a modal dialog, the user should get an inline notification informing them of the result.
+(If a modal dialog has more than 2 screens, follow the pattern defined for wizards)
+![Inline notification for an error](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/errorPlacement.png?raw=true)
+*Error placement*
+
+### Exiting modals
+
+Modals that have only 1 step, and are fairly simple can be cancelled in a few ways:
```
Comment (kruai) [inline src/docs/uxd/alertsNotifications/DOC.md]: I'd either remove the comma or put another one after "fairly simple".

File: src/docs/uxd/alertsNotifications/DOC.md [hunk 8]
```diff
@@ -1,5 +1,87 @@
...
+- Clicking "Cancel"
+- NOT clicking outside the modal
+
+In some cases, you can use a cancellation alert to inform the user that they will lose their data if cancelling the process
+![Example of cancellation alert](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/cancellationAlert.png?raw=true)
+*Example of cancellation alert*
+
+## Wizards
+
+Wizards are multiple step processes. In RHCS, these mostly occur in modal dialogs. At the end of a wizard, show a confirmation screen.
```
Comment (kruai) [inline src/docs/uxd/alertsNotifications/DOC.md]: "multiple step processes" -> "multiple-step processes"

File: src/docs/uxd/alertsNotifications/DOC.md [hunk 9]
```diff
@@ -1,5 +1,87 @@
...
+![Wizard with confirmation screen](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/wizardConfirmation.png?raw=true)
+*Wizard with confirmation screen*
+
+After the confirmation screen. there are a few different options:
```
Comment (kruai) [inline src/docs/uxd/alertsNotifications/DOC.md]: Change period into a comma

File: src/docs/uxd/alertsNotifications/DOC.md [hunk 10]
```diff
@@ -1,5 +1,87 @@
...
+
+**A process that does not take significant time**
+Close modal and show results. Show inline notification (see below for more details)
+If failure, see error state below.
+
+**A process that takes time, and must complete before proceeding**
+Show progress indicator as an additional screen in the modal.
+At the end of the progress, close the modal automatically and show an inline notification (see below for more information)
```
Comment (kruai) [inline src/docs/uxd/alertsNotifications/DOC.md]: "end of the progress" -> "end of the process"?

File: src/docs/uxd/toolbarsFilters/DOC.md [hunk 11]
```diff
@@ -1,5 +1,35 @@
 # Toolbars & Filters
 
-Content TBD
+![Toolbar layout](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/toolbarLayout.png?raw=true)
 
-[Google Doc](https://docs.google.com/document/d/1Uq1cmLxrUBhhDZrUe7KIdesLKORT2IF5M13XqCJ6yYY/edit?usp=sharing)
+1. **Filtering**: Filtering should be on the left-most side of the toolbar. All lists should have some filtering. (See filtering section for more details)
+2. **Action buttons**: Arrange action buttons to the right of the filtering. There can be 1 primary action (an action is not required), 1 secondary action, and any overflow goes into the kebab.
+3. **Kebab & Export**: Export should always go into the kebab, even if it is the only action on the table or list.
+4. **Pagination**: Use pagination on top and bottom of table. [PF4 pagination](https://www.patternfly.org/v4/documentation/react/components/pagination/) will responsively react to show only the results count when in smaller spaces. This is the only results count shown. (See filtering section for more details about how the results count works with filtering)
```
Comment (kruai) [inline src/docs/uxd/toolbarsFilters/DOC.md]: Suggestion: "Use pagination on the top and bottom of the table."

**Review by kruai — DISMISSED** [review]
I have some minor suggestions, but I think this would be fine with or without them.

File: src/docs/uxd/alertsNotifications/DOC.md (hunk 10 above)
Comment (ryelo) [reply src/docs/uxd/alertsNotifications/DOC.md]: it should read progress indicator

It's referencing the step above

**Review by kruai — APPROVED** [review]

**Review by karelhala — APPROVED** [review]

**Pull request merged**